engines/sci/graphics/frameout.cpp

```diff
@@ -1334,14 +1366,15 @@ void GfxFrameout::kernelSetNowSeen(const reg_t screenItemObject) const {
 
 	ScreenItem *screenItem = plane->_screenItemList.findByObject(screenItemObject);
 	if (screenItem == nullptr) {
-		error("kSetNowSeen: Screen item %04x:%04x not found in plane %04x:%04x", PRINT_REG(screenItemObject), PRINT_REG(planeObject));
+		return false;
```

Comment:
why is a non existing object now ignored inside kSetNowSeen?
in sci32 that (also) results in a fatal error, so the original code was accurate.